pelican/tests/test_contents.py

```diff
@@ -497,6 +497,30 @@ def test_slugify_tags_with_dots(self):
         article = Article(**article_kwargs)
         self.assertEqual(article.url, 'fedora.qa/this-week-in-fedora-qa/')
 
+    def test_check_save_as_detects_breakout(self):
+        settings = get_settings()
+        settings['TAG_SUBSTITUTIONS'] = [('Fedora QA', 'fedora.qa', True)]
+        settings['ARTICLE_URL'] = '{tag}/{slug}/'
+        article_kwargs = self._copy_page_kwargs()
+        article_kwargs['metadata']['tag'] = Tag('Fedora QA', settings)
```

Comment:
Uhh, please no. I'm guessing you copied it from the test above but neither `tag` metadata nor `{tag}` formatting in the `*_URL/*_SAVE_AS` is a thing for pelican. There is only `tags` metadata that has special meaning for pelican and since it's assumed to contain multiple values, it's not considered for formatting.

If you want to have something like this, please use `category` instead. The test above and the `tag` references was a "mistake" and will be fixed (when i have time :) ).

Same for the test below.